src/main/java/com/ss/training/lms/controller/AdminBorrwerController.java

```diff
@@ -0,0 +1,118 @@
+package com.ss.training.lms.controller;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import com.ss.training.lms.entity.Borrower;
+import com.ss.training.lms.service.admin.AdminBorrowerService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AdminBorrwerController {
+
+    @Autowired
+    AdminBorrowerService adminBorrowerService;
+
+    @PostMapping(path = "/lms/admin/borrowers/add")
+    public ResponseEntity<Borrower> addABorrower(@RequestBody Borrower borrower) {
+        HttpStatus status = HttpStatus.CREATED;
+
+        if (borrower == null || borrower.getCardNo() == null || borrower.getName() == null) {
```

Comment:
The database schema does allow borrower names to be null, but if we want to prevent users from creating null borrower names anyhow then I don't see anything wrong with that.